trunk/SUAVE/Components/Energy/Converters/Propeller.py

```diff
@@ -693,87 +693,4 @@ def spin_variable_pitch(self,conditions):
         
         return thrust, torque, power, Cp, outputs  , etap  
 
-    
-    def spin_surrogate(self,conditions):
```

Comment:
any reason for the removal of the surrogate?

Reply:
Yes, We don't have any regression test for it so I will work in it when I do the BEMT PR.